content/en/docs/tasks/run-application/scale-stateful-set.md

```diff
@@ -72,32 +75,29 @@ kubectl patch statefulsets <stateful-set-name> -p '{"spec":{"replicas":<new-repl
 
 ## Troubleshooting
 
-### Scaling down doesn't work right
+### Scaling down does not work right
 
 You cannot scale down a StatefulSet when any of the stateful Pods it manages is unhealthy. Scaling down only takes place
 after those stateful Pods become running and ready.
 
-With a StatefulSet of size > 1, if there is an unhealthy Pod, there is no way
-for Kubernetes to know (yet) if it is due to a permanent fault or a transient
-one (upgrade/maintenance/node reboot). If the Pod is unhealthy due to a permanent fault, scaling
+With a StatefulSet size of greater than one, currently Kubernetes is unable to determine if an unhealthy Pod is a result of a permanent fault or a transient fault (upgrade/maintenance/node reboot).
```

Comment:
This is a tough one, but we can get it simpler and clearer still.

If `spec.replicas` > 1, Kubernetes cannot determine the reason for an unhealthy Pod. It might be the result of a permanent fault or of a transient fault. A transient fault can be caused by a restart required by upgrading or maintenance.

(I'm not sure how useful this paragraph is, either. Note that StatefulSet does not have a property called "size," and it's not clear what to do or how to monitor to remedy the situation. If you can investigate further, that would be awesome -- fine to go with what's here for now and submit a separate PR later for technical clarification.)

Reply:
Done